images-list

```diff
@@ -10,6 +10,7 @@ banzaicloud/logging-operator rancher/banzaicloud-logging-operator 3.6.0
 banzaicloud/logging-operator rancher/banzaicloud-logging-operator 3.7.0
```

Comment:
no fluentD or fluentBit bumps ?

Reply:
Nope, I checked too. They are the same as 3.8.0.